### DISCUSSION TITLE
Issue form preview displays "About" instead of "Description"

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#5111] [Projects and Issues]
When looking at the preview of a issue form (Directly viewing the YAML file) does the top table section display `About` for the `description` key, which can be misleading.

Preview:
![image](https://user-images.githubusercontent.com/11576465/129482509-2df13d00-3abd-4616-9a4d-b469c4af4792.png)

Raw:
![image](https://user-images.githubusercontent.com/11576465/129482520-8289c4d7-02fa-4ee1-a07a-57f9189846a2.png)

**Comment by lukehefson** [comment]
Agree that this is odd. Thanks for reporting @Andre601!
